crates/libcontainer/src/process/container_main_process.rs

```diff
@@ -75,15 +75,6 @@ pub fn container_main_process(container_args: &ContainerArgs) -> Result<(Pid, bo
         })
     };
 
-    // Before starting the intermediate process, mark all non-stdio open files as O_CLOEXEC
-    // to ensure we don't leak any file descriptors to the intermediate process.
-    // Please refer to https://github.com/opencontainers/runc/security/advisories/GHSA-xr7r-f8xq-vfvv for more details.
-    let syscall = container_args.syscall.create_syscall();
-    syscall.close_range(0).map_err(|err| {
```

Comment:
I believe this revert is correct. Reading the security advisory, the action called out in the advisory is to set `O_CLOEXEC` on all FDs other than stdio. The `close_range` in the `--preserve-fds` check in the init process should take care of all this case.